apps/etterlatte-behandling/src/main/kotlin/brev/BrevRoute.kt

```diff
@@ -0,0 +1,86 @@
+package no.nav.etterlatte.brev
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.put
+import io.ktor.server.routing.route
+import no.nav.etterlatte.libs.common.feilhaandtering.krevIkkeNull
+import no.nav.etterlatte.libs.ktor.route.BEHANDLINGID_CALL_PARAMETER
+import no.nav.etterlatte.libs.ktor.route.behandlingId
+import no.nav.etterlatte.libs.ktor.route.sakId
+import no.nav.etterlatte.libs.ktor.token.brukerTokenInfo
+import org.slf4j.LoggerFactory
+import kotlin.time.DurationUnit
+import kotlin.time.measureTimedValue
+
+fun Route.brevRoute(service: BrevService) {
+    val logger = LoggerFactory.getLogger("BrevRoute")
+
+    route("api/behandling/brev/{$BEHANDLINGID_CALL_PARAMETER}/vedtak") {
+        post {
+            // TODO tilgangsjekk?
```

Comment:
Ja, disse burde wrappes med kunSaksbehandler + riktig setting av skrivetilgang. Du har vel noe ut av boksen ved å bruke `BEHANDLINGID_CALL_PARAMETER`, men for skrivetilgang må vi være strengere og det er greit å ha det eksplisitt